includes/widgets/playing-cards.php

```diff
@@ -0,0 +1,404 @@
+		return [ 'general' ];
+	}
+
+	/**
+	 * Get widget keywords.
+	 *
+	 * Retrieve the list of keywords the widget belongs to.
+	 *
+	 * @since 3.12.1
+	 * @access public
+	 *
+	 * @return array Widget keywords.
+	 */
+	public function get_keywords(): array {
+		return [ 'card' ];
+	}
+
+	/**
+	 * Register playing cards widget controls.
+	 *
+	 * Adds different input fields to allow the user to change and customize the widget settings.
+	 *
+	 * @since 3.12.1
+	 * @access protected
+	 */
+	protected function register_controls() {
+		$this->start_controls_section(
+			'section_cards',
+			[
+				'label' => esc_html__( 'Cards', 'elementor' ),
+				'tab' => Controls_Manager::TAB_CONTENT,
+			]
+		);
+
+		$repeater = new Repeater();
+
+		$repeater->add_control(
+			'card_type',
+			[
+				'label' => esc_html__( 'Type', 'elementor' ),
+				'type' => Controls_Manager::SELECT,
+				'options' => [
+					'spade' => esc_html__( 'Spade', 'elementor' ),
+					'diamond' => esc_html__( 'Diamond', 'elementor' ),
+					'heart' => esc_html__( 'Heart', 'elementor' ),
+					'club' => esc_html__( 'Club', 'elementor' ),
+				],
+				'default' => 'spade',
+				'separator' => 'before',
+			]
+		);
+
+		$repeater->add_control(
+			'card_value',
+			[
+				'label' => esc_html__( 'Value', 'elementor' ),
+				'type' => Controls_Manager::SELECT,
+				'options' => [
+					'2'  => '2',
+					'3'  => '3',
+					'4'  => '4',
+					'5'  => '5',
+					'6'  => '6',
+					'7'  => '7',
+					'8'  => '8',
+					'9'  => '9',
+					'10' => '10',
+					'J'  => esc_html_x( 'J', 'Playing cards value', 'elementor' ),
+					'Q'  => esc_html_x( 'Q', 'Playing cards value', 'elementor' ),
+					'K'  => esc_html_x( 'K', 'Playing cards value', 'elementor' ),
+					'A'  => esc_html_x( 'A', 'Playing cards value', 'elementor' ),
+				],
+				'default'   => '2',
+				'separator' => 'before',
+			]
+		);
+
+		$this->add_control(
+			'cards_list',
+			[
+				'label' => esc_html__( 'Hand', 'elementor' ),
+				'type' => Controls_Manager::REPEATER,
+				'fields' => $repeater->get_controls(),
+				'default' => [],
+				'title_field' => '{{ card_type }} ({{ card_value }})',
+			]
+		);
+
+		$this->add_control(
+			'hover_animation',
+			[
+				'label' => esc_html__( 'Hover Animation', 'elementor' ),
+				'type' => Controls_Manager::SELECT,
+				'options' => [
+					'pulse' => esc_html__( 'Pulse', 'elementor' ),
+					'bounce' => esc_html__( 'Bounce', 'elementor' ),
+				],
+				'default' => 'pulse',
+				'separator' => 'before',
+			]
+		);
+
+		$this->end_controls_section();
+
+		$this->start_controls_section(
+			'section_card_styles',
+			[
+				'label' => esc_html__( 'Card', 'elementor' ),
+				'tab' => Controls_Manager::TAB_STYLE,
+			]
+		);
+
+		$this->add_control(
+			'card_background',
+			[
+				'label' => esc_html__( 'Background Color', 'elementor' ),
+				'type' => Controls_Manager::COLOR,
+				'selectors' => [
+					'{{WRAPPER}} .elementor-playing-card' => 'background-color: {{VALUE}};',
```

Comment:
```suggestion
					'{{WRAPPER}}' => '--e-playing-card-bg-color: {{VALUE}};',
```
For all styling controls @pkniazevych